### PR TITLE
Doc cfg attr

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Follow-up of https://github.com/gtk-rs/gir/pull/988

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
@bilelmoussaoui Running with `--all-features` on stable is going to be problematic with this. Should I disable it for the stable channel in here?

**Comment by bilelmoussaoui** [comment]
I guess we will have to manually switch to the highest feature there's?

File: pangocairo/src/lib.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT license, see the LICENSE file or <https://opensource.org/licenses/MIT>
 
 #![allow(deprecated)]
+#![cfg_attr(feature = "dox", feature(doc_cfg))]
```
Comment (sdroege) [inline pangocairo/src/lib.rs:6]: This only works with nightly or not?

File: gdk-pixbuf/src/pixbuf.rs [hunk 2]
```diff
@@ -344,6 +344,7 @@ impl Pixbuf {
     }
 
     #[cfg(any(feature = "v2_32", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_32")))]
```
Comment (sdroege) [inline gdk-pixbuf/src/pixbuf.rs:347]: Do you have a script to update all these?

File: pangocairo/src/lib.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply pangocairo/src/lib.rs:6]: Only for nightly, which is why I added this line.

File: gdk-pixbuf/src/pixbuf.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply gdk-pixbuf/src/pixbuf.rs:347]: No, by hand.

File: pangocairo/src/lib.rs (hunk 1 above)
Comment (sdroege) [reply pangocairo/src/lib.rs:6]: If someone wants to build docs locally that's a bit annoying, but I guess acceptable :)

**Comment by GuillaumeGomez** [comment]
@bilelmoussaoui I fixed the CI. Do you think it's fine as is?

**Comment by bilelmoussaoui** [comment]
@GuillaumeGomez do we have to disable running tests/builds on stable/beta/v1.40 ? otherwise the `if: ....` parts should be removed as you have switched to the latest feature there's for each crate already

**Comment by GuillaumeGomez** [comment]
The feature I added is nightly only, which means we can't run with `dox` nor `--all-features`. Which is why I splitted both runs where `--all-features` was used.

**Comment by sdroege** [comment]
@GuillaumeGomez What's missing here?

**Comment by GuillaumeGomez** [comment]
The confirmation from @bilelmoussaoui I guess?

**Comment by bilelmoussaoui** [comment]
Let's wait till the CI can work again

**Pull request merged**